fuse/index.d.ts

```diff
@@ -1,30 +1,36 @@
 // Type definitions for Fuse.js 2.5.0
 // Project: https://github.com/krisk/Fuse
-// Definitions by: Greg Smith <https://github.com/smrq/>
+// Definitions by: Alan Agius <https://github.com/alan-agius4/>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-declare class Fuse {
-	constructor(list: any[], options?: fuse.IFuseOptions);
-	search(pattern: string): any[];
-}
+declare module 'fuse.js' {
```

Comment:
If you're declaring the "fuse.js" module, move everything to a "fuse.js" directory and don't bother with this wrapper.